layouts/shortcodes/featured/ballyhoo.html

```diff
@@ -0,0 +1,9 @@
+{{ partial "tw/ballyhoo.html" (dict
```

Comment:
Should we have this get the parameters from the short code attributes instead of hard coding them?

Reply:
I was thinking this would keep it simple since this is a one-off, but now I realize we should go ahead and make it complex in case we want to use the same shortcode to promote other stories/tools in the future -- https://github.com/spotlightpa/poor-richard/pull/631#discussion_r1565837714